conf/command.conf

```diff
@@ -0,0 +1,65 @@
+# CustomCommand section must be defined before Latency monitor to support all commands
+#
+# NOTE: only support maximum 16 custom commands
+
+CustomCommand {
+    # from redis source src/modules/hello.c
+    # hello.push.native key value
+    hello.push.native {  # name of command, must be lowercase.
+                         # key parameter is located at first argument. i.e. command key ...
+                         # Currently support only 1 key with first argument.
+        minArgs 3        # minimum arguments(including command itself), default is 2(key only)
+        maxArgs 3        # maximum arguments(including command itself), default is 2(key only)
+        mode Write       # a command mode Read/Write/Admin is exclusive, default is Write
+    }
+    # hello.repl2 <list-key>
```

Comment:
all cases mode is **Write**, give a case with **mode Read** is better

Reply:
In case of 'Write' , @antirez used RedisModule_OpenKey with REDISMODULE_READ|REDISMODULE_WRITE.

https://github.com/antirez/redis/blob/8ac7af1c5d4d06d6c165e35d67a3a6a70e5d98c3/src/modules/hellotype.c#L107

So do I. But mostly the command mode should not be Write. I'll check.

Reply:
just give a case for example,